fhirclient/server.py

```diff
@@ -184,6 +187,7 @@ def _get(self, path, headers={}, nosign=False):
         
         # perform the request but intercept 401 responses, raising our own Exception
         res = self.session.get(url, headers=headers)
+        logging.debug('SMART SERVER:\r\n{0}'.format(dump.dump_response(res).decode('utf-8')))
```

Comment:
I will change to os.linesep. I think that is right way of doing..
The logger does not give me the flexibility to debug fhir client only.. I have to change the settings at the package level for them to work as expected, hence I had to change the logger..

I will take care of separate PR per issue.. I tried creating a separate one but it merged into the current PR.. May be I have to submit a new PR only after the current one is taken care of..